packages/sdk/src/lib/dataProtectorSharing/subscribeToCollection.ts

```diff
@@ -41,10 +46,14 @@ export const subscribeToCollection = async ({
 
   try {
     const { txOptions } = await iexec.config.resolveContractsClient();
-    const tx = await sharingContract.subscribeTo(vCollectionTokenId, {
-      value: collectionDetails.subscriptionParams.price,
-      ...txOptions,
-    });
+    const tx = await sharingContract.subscribeTo(
+      vCollectionTokenId,
+      vDuration,
```

Comment:
Why do we need to pass this `duration` here?
The smart-contract should know the duration, as it has been set by the collection owner?

Reply:
in order to avoid front run